tests/utils/test_utils.py

```diff
@@ -68,6 +69,27 @@ def test_move_to_device(caplog):
     assert move_to_device((torch.tensor([1, 2]), torch.tensor([3, 4])), -1)
 
 
+def test_merge_objects(caplog):
+    """Unit test of merge_objects."""
+    caplog.set_level(logging.INFO)
+
+    assert torch.equal(
+        merge_objects(torch.Tensor([1, 2]), torch.Tensor([2, 3])),
+        torch.Tensor([1, 2, 2, 3]),
```

Comment:
This should be `torch.Tensor[[1, 2], [2, 3]]`? We want to merge based on the first dim.